apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UrlSigner.Options.cs

```diff
@@ -47,18 +47,73 @@ public sealed class Options
             /// </summary>
             public SigningVersion SigningVersion { get; }
 
-            private Options(TimeSpan? duration, DateTimeOffset? expiration, SigningVersion version)
```

Comment:
For SigningVersion, given the rework of this implementation, could you update to default to V4?

Reply:
Done